do/loadbalancers.go

```diff
@@ -179,6 +184,126 @@ func (l *loadbalancers) EnsureLoadBalancer(clusterName string, service *v1.Servi
 
 }
 
+func checkIfPortInRange(port int, fwPortRange string) bool {
+
+	fwPortList := strings.Split(fwPortRange, "-")
```

Comment:
I had a look at this, the only issue is that the first value can sometimes be a string containing "all", rather than just two integers. Need to think about how to clean this up.

Reply:
kind : Service 
apiVersion : v1 
metadata :
   name : bonjour 
  annotations :
     service.beta.kubernetes.io/do-loadbalancer-certificate-id : " 1234-5678-9012-3456 "
     service.beta.kubernetes.io/do-loadbalancer- protocole : " https " 
    # Décommentez une fois que hello.example.com pointe vers l'adresse IP externe de l'équilibreur de charge DO. 
    # service.beta.kubernetes.io/do-loadbalancer-hostname: "hello.example.com" 
spec :
   type : LoadBalancer 
  sélecteur :
    app : hello 
  ports :
    - Nom : https 
      protocole : TCP 
      Port : 443 
      targetPort : 80
---
apiVersion : apps / v1 
kind : Métadonnées de déploiement
 :
   nom : bonjour spec :
   répliques : 3 sélecteur :
     matchLabels :
       app : bonjour modèle :
     metadata :
       labels :
         app : bonjour spec :
       conteneurs :

  
  
    
      - nom : bonjour 
        image : ports snormore / bonjour
         :
        - protocole containerPort : 80
           : TCP